common/build-style/cmake.sh

```diff
@@ -18,6 +19,20 @@ do_configure() {
 			ppc*) _CMAKE_SYSTEM_PROCESSOR=powerpc ;;
 			*) _CMAKE_SYSTEM_PROCESSOR=generic ;;
 		esac
+		case "$XBPS_TARGET_MACHINE" in
+			x86_64*) _MACHINE=x86_64 ;;
+			i686) _MACHINE=i386 ;;
+			aarch64*) _MACHINE=aarch64 ;;
+			armv*) _MACHINE=arm ;;
+			mipsel*) _MACHINE=mipsel ;;
+			mips*) _MACHINE=mips ;;
+			ppc64le*) _MACHINE=ppc64le ;;
+			ppc64*) _MACHINE=ppc64 ;;
+			ppc*) _MACHINE=ppc ;;
+			*)
+				msg_error "unknown qemu architecture: $XBPS_TARGET_MACHINE"
+			;;
+		esac
```

Comment:
Since we're using this in multiple places (e.g. the gir build_helper), could we make this generally available as  `QEMU_ARCH` or `QEMU_MACHINE` maybe?

Reply:
I added it as `XBPS_TARGET_QEMU_MACHINE` for now.

Reply:
Thanks :)